linux/user-and-file-management/linux-intro/hello-linux.md

```diff
@@ -0,0 +1,51 @@
+---
+author: kapnobatai136
+
+aspects:
+  - introduction
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - linux
+
+links:
+  - '[Install Ubuntu](https://ubuntu.com/tutorials/tutorial-install-ubuntu-desktop#1-overview){website}'
+  - '[Try Ubuntu](https://ubuntu.com/tutorials/try-ubuntu-before-you-install#1-getting-started){website}'
+
+---
+
+# Hello Linux
+
+---
+## Content
+
+All this talk, and we still haven't shown you how to use Linux 😅.
+
+> Note: we're going to assume you have an installed Linux distribution. If you don't, check out the *Learn More* section for Ubuntu installation guides.
+
+![printing-hello-linux-in-shell](https://img.enkipro.com/b293f5a296e067d80f9dc7e77c7072d3.gif)
+
+This is the Linux terminal[1]! It's where you'll be typing most of the commands.
+
+In computer science, what you see in the example is called *printing*, a fancy word for displaying text. 
```

Comment:
Maybe more precise to talk about the command? Especially since we mention it right before? 

Also, the idea of commands is much more important here than the detail that displaying text in a terminal is called "printing":

```suggestion
What you see in the example is how to use the `echo` command to display some text. 
```